index.html

```diff
@@ -5,9 +5,357 @@
+            </nav> 
+            <div class="hero-content space-between">
+                <div class="hero-content-txt">
+                    <h1 class="hero-title white title-text">
+                        Reach more <br> people in real time
+                    </h1>
+                    <p class="hero-para white paragraph-text">
+                        The screeno ecosystem is designed to help you generate profit. Set up complete sales and marketing funnels with ease using the Screeno.
+                    </p>
+                    <div class="hero-cta centrally-aligned space-between">
+                        <button class="blue-background button">
+                            Start Free Trial
+                        </button>
+                        <button class="hero-cta-button white button">
+                            Login
+                        </button>
+                    </div>
+                </div>
+                <div class="hero-content-img">
+                    <img src="img/Main-TV-frame.png" alt="zdjęcię przedstawiające telewizor">
+                    <img class="hero-img-phone" src="img/Main-phone-frame.png" alt="zdjęcię przedstawiające telefon">
+                    <div class="hero-absolute-box flex-center">
+                        <img class="hero-live" src="img/live.png" alt="na żywo">
+                    </div>
+                </div>
+            </div>
+            <ul class="space-between hero-image-list">
+                <li class="hero-platforms">
+                    <img src="img/tv.png" alt="napis apple-tv">
+                </li>
+                <li class="hero-platforms">
+                    <img src="img/android.png" alt="napis android-tv">
+                </li>
+                <li class="hero-platforms">
+                    <img src="img/appleexport.png" alt="napis apple-watch">
+                </li>
+                <li class="hero-platforms">
+                    <img src="img/iphone.png" alt="napis iphone">
+                </li>
+                <li class="hero-platforms">
+                    <img src="img/androidicon.png" alt="napis android">
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="flex-center">
+        <div class="section-layout">          
+            <h2 class="tile-section-title dark text-center title-text">
+                All-in-one platform
+            </h2>
+            <p class="subtxt text-center dark paragraph-text">
+                You take care of the video quality and we take care of <br> everything else.
+            </p>
+            <div class="tile-section-cta centrally-justified">
+                <button class="tile-left-button blue-background button">
+                    Start Free Trial
+                </button>
+                <button class="tile-right-button dark button">
+                    Login
+                </button>
+            </div>
+            <div class="tile-list-box">
```

Comment:
ten element jest niepotrzebny, już to ul wystarczy ostylować nie trzeba tego opakowywać jeszcze w diva